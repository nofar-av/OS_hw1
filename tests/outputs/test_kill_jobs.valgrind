==28811== Memcheck, a memory error detector
==28811== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28811== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28811== Command: /home/omer/OS_wet1/smash
==28811== Parent PID: 28807
==28811== 
==28811== 
==28811== FILE DESCRIPTORS: 4 open at exit.
==28811== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_kill_jobs.valgrind
==28811==    <inherited from parent>
==28811== 
==28811== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_kill_jobs.err
==28811==    <inherited from parent>
==28811== 
==28811== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_kill_jobs.out
==28811==    <inherited from parent>
==28811== 
==28811== Open file descriptor 0:
==28811==    <inherited from parent>
==28811== 
==28811== 
==28811== HEAP SUMMARY:
==28811==     in use at exit: 0 bytes in 0 blocks
==28811==   total heap usage: 155 allocs, 155 frees, 89,067 bytes allocated
==28811== 
==28811== All heap blocks were freed -- no leaks are possible
==28811== 
==28811== For counts of detected and suppressed errors, rerun with: -v
==28811== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
