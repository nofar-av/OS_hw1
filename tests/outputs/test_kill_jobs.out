smash> submitted by Eran
smash> smash> smash> smash> [1] ./printSignals.exe & : 5818 0 secs
[2] sleep 200& : 5819 0 secs
[3] sleep 300& : 5820 0 secs
smash> signal number 9 was sent to pid 5819
smash> smash> [1] ./printSignals.exe & : 5818 1 secs
[3] sleep 300& : 5820 1 secs
[4] sleep 400& : 5935 0 secs
smash> smash> smash> smash> smash> smash> signal number 9 was sent to pid 5935
smash> signal number 5 was sent to pid 5818
smash> received signal number: 5
